zed_wrapper/params/common.yaml

```diff
@@ -79,16 +79,16 @@ sensors:
     publish_imu_tf:             true                            # publish `IMU -> <cam_name>_left_camera_frame` TF
 
 object_detection:
-    od_enabled:                 false                           # True to enable Object Detection [not available for ZED]
-    model:                      0                               # '0': MULTI_CLASS_BOX - '1': MULTI_CLASS_BOX_ACCURATE
-    confidence_threshold:       50                              # Minimum value of the detection confidence of an object [0,100]
-    max_range:                  15.                             # Maximum detection range
-    object_tracking_enabled:    true                            # Enable/disable the tracking of the detected objects
-    mc_people:                  true                            # Enable/disable the detection of persons for 'MULTI_CLASS_BOX_X' models
-    mc_vehicle:                 true                            # Enable/disable the detection of vehicles for 'MULTI_CLASS_BOX_X' models
-    mc_bag:                     true                            # Enable/disable the detection of bags for 'MULTI_CLASS_BOX_X' models
-    mc_animal:                  true                            # Enable/disable the detection of animals for 'MULTI_CLASS_BOX_X' models
-    mc_electronics:             true                            # Enable/disable the detection of electronic devices for 'MULTI_CLASS_BOX_X' models
-    mc_fruit_vegetable:         true                            # Enable/disable the detection of fruits and vegetables for 'MULTI_CLASS_BOX_X' models
-    mc_sport:                   true                            # Enable/disable the detection of sport-related objects for 'MULTI_CLASS_BOX_X' models
-
+    od_enabled:                         false                           # True to enable Object Detection [not available for ZED]
+    model:                              'MULTI_CLASS_BOX_ACCURATE'      # 'MULTI_CLASS_BOX_FAST', 'MULTI_CLASS_BOX_MEDIUM', 'MULTI_CLASS_BOX_ACCURATE', 'PERSON_HEAD_BOX_FAST', 'PERSON_HEAD_BOX_ACCURATE'
+    max_range:                          15.                             # Maximum detection range
+    allow_reduced_precision_inference:  true                            # Allow inference to run at a lower precision to improve runtime and memory usage
+    prediction_timeout:                 0.5                             #  During this time [sec], the object will have OK state even if it is not detected. Set this parameter to 0 to disable SDK predictions            
```

Comment:
I've set this parameter to 0, and I am still able to see object detection predictions, which is not what is explained by this comment.

Reply:
That's an SDK behavior :(

Reply:
Will report this to the SDK team :+1: